release/staging-resources-cdk/lib/StagingResourcesStack.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {Stack, StackProps} from 'aws-cdk-lib';
+import {Construct} from 'constructs';
+import {CfnPublicRepository} from 'aws-cdk-lib/aws-ecr'
+
+export class StagingResourcesStack extends Stack {
```

Comment:
I know this PR is mainly boilerplate, but it would be kind of nice to have a comment here briefly summarizing which resources this stack will create, similar to how you summarized the purpose of the `GithubAccessStack`. It's not blocking or anything though.

Reply:
Thanks! I have some follow-on PRs and will add then.